### PR TITLE
Add iforgor config and readme

### PR DESCRIPTION
**Pull request by nanocryk**
Add config file and README for tool `iforgor`, which will allow us to share scripts for commands commonly run when working on Tanssi.

### CODE REVIEW

File: tools/iforgor/tanssi.toml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+script = "git fetch origin && git pull"
+
+[[entries]]
+name = "[Git] Push"
+script = "git push"
+
+[[entries]]
+name = "[Git] Delete local branches (select)"
+script = """
+git branch | ichoose --multi --title "Select branches to delete" | while read x ; do git branch -D $x ; done
```
Comment (ParthDesai) [inline tools/iforgor/tanssi.toml:188]: `ichoose` does not exists on macos. Is it built-in for linux?

File: tools/iforgor/tanssi.toml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+echo "\nℹ️ Running 'cargo fmt'\n"
+cargo fmt
+
+echo "\nℹ️ Running 'toml-maid'\n"
+toml-maid
+
+echo "\nℹ️ Running 'zepter'\n"
+zepter
+
+cd test
```
Comment (ParthDesai) [inline tools/iforgor/tanssi.toml:159]: You might want to use `pushd` and `popd` here. So, that user is back in the main directory once the command is run instead in the test directory.

File: tools/iforgor/tanssi.toml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+# Typescript
+
+## API
+
+[[entries]]
+name = "[Tanssi] Generate TS-API"
+only_in_dir = "**/tanssi"
+script = """
+cd typescript-api
```
Comment (ParthDesai) [inline tools/iforgor/tanssi.toml:114]: Same here. Use `pushd` and `popd` so user is returned to the same directory.

File: tools/iforgor/tanssi.toml (hunk 1 above)
Comment (nanocryk) [reply tools/iforgor/tanssi.toml:188]: It is another binary similar to iforgor. I talk about it in the README.

File: tools/iforgor/tanssi.toml (hunk 2 above)
Comment (nanocryk) [reply tools/iforgor/tanssi.toml:159]: Directory change doesn't persist after script has finished running. Script is run in a new bash process and doesn't affect the current directory of the user.

File: tools/iforgor/tanssi.toml (hunk 3 above)
Comment (ParthDesai) [reply tools/iforgor/tanssi.toml:114]: *disregard*

File: tools/iforgor/tanssi.toml (hunk 1 above)
Comment (ParthDesai) [reply tools/iforgor/tanssi.toml:188]: Okay. So, both needs to be installed on both platform, correct?

**Review by girazoki — APPROVED** [review]

**Pull request merged**